src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3889,6 +3927,28 @@ void ApplicationManagerImpl::OnPTUFinished(const bool ptu_result) {
 #endif  // SDL_REMOTE_CONTROL
 }
 
+void ApplicationManagerImpl::PutDriverDistractionMessageToPostponed(
+    ApplicationSharedPtr application) const {
+  LOG4CXX_AUTO_TRACE(logger_);
+  if (hmi_apis::Common_DriverDistractionState::INVALID_ENUM ==
+      distracting_driver_state()) {
+    LOG4CXX_INFO(logger_, "DriverDistractionState is INVALID_ENUM");
```

Comment:
@LitvinenkoIra please do not use INFO level loggin for such logs. Consider https://github.com/smartdevicelink/sdl_core/wiki/SDL-Logging-levels

Reply:
@LuxoftAKutsan done 212bee04d27efd4be438aa8b57966dee010fda8e